tests/test_timm_wrap.py

```diff
@@ -4,15 +4,21 @@
 # LICENSE file in the root directory of this source tree.
 
 import pytest
-import timm
 import torch
-from timm.models.vision_transformer import VisionTransformer
+
+try:
+    import timm
+    from timm.models.vision_transformer import VisionTransformer
+except ImportError:
+    timm = None
+    VisionTransformer = None
```

Comment:
Let's not make not having `timm` a hard failure in our tests

Reply:
still required given the timm dependency fix ? (pulling timm from main)

Reply:
I've added `timm` back in the `requirements` file just to continue testing it as before (and be the least BC-breaking as possible). But this now works even if we remove it, which was the original goal when I did this.

I can revert those changes if you want, but I still think keeping test dependencies small is a plus. Up to you